packages/react/src/components/AccountSettings/DeleteUser/DeleteUser.tsx

```diff
@@ -21,6 +22,15 @@ function DeleteUser({
   const [state, setState] = React.useState<DeleteUserState>('IDLE');
   const [errorMessage, setErrorMessage] = React.useState<string | null>(null);
 
+  React.useEffect(() => {
+    const clearUserAgent = setUserAgent({
```

Comment:
Headers
<img width="585" alt="Screenshot 2023-10-23 at 6 05 17 PM" src="https://github.com/aws-amplify/amplify-ui/assets/70536670/7ef72662-ead4-4a4f-b6fb-44bea4e4f028">

Reply:
Should this be 'AccountSettings' rather than 'Authenticator'?

Reply:
Actually this looks correct in the code, the screenshot is probably from a different request

Reply:
My bad 
<img width="786" alt="DeleteUser" src="https://github.com/aws-amplify/amplify-ui/assets/70536670/cd038a3d-8c8f-43ec-ae99-ef6d2daa19dd">